server/routes/index.ts

```diff
@@ -173,15 +173,7 @@ export default function routes(services: Services): Router {
         page: Page.WorkAndSkills,
       })
 
-      /* New Work & Skills page introduced in epic RR-575
-         Render the new view if the NEW_WORK_AND_SKILLS_TAB_ENABLED feature toggle is enabled
-         TODO - The feature toggle and this ternary will be removed in RR-607
-      */
-      const viewTemplate = config.featureToggles.newWorkAndSkillsTabEnabled
-        ? 'pages/new_workAndSkills'
-        : 'pages/workAndSkills'
-
-      res.render(viewTemplate, {
+      res.render('pages/workAndSkills', {
```

Comment:
Whilst we had the feature toggle in place, the "new" views were all prefixed with `new_`
As well as removing the feature toggle and logic based on it, this PR also deletes the original views (`workAndSkills`) and renames the `new_` views to remove the `new_` prefix